node/validation/manifest/schema.ts

```diff
@@ -35,7 +34,7 @@ const functionConfigSchema = {
         type: 'string',
         format: 'regexPattern',
         errorMessage:
-          'excluded_patterns needs to be an array of regex that starts with ^ followed by / and ends with $ without any additional slashes before and afterwards',
+          'excluded_patterns must be an array of regex that starts with ^/ and ends with $ (e.g. ^/blog/[d]{4}$)',
```

Comment:
If this is meant to be a customer-facing error message, I think it would be helpful to use the customer-facing input name rather than the name of the internal property we validate. Using the customer-facing name should help folks find and fix the invalid value that's triggering the error message.

```suggestion
          'excludedPattern must be an array of regex that starts with ^/ and ends with $ (e.g. ^/blog/[d]{4}$)',
```

Reply:
Hmm.. The worst part about this validation is that it will most likely not be fixable by the user. 

1. If a user misses the slash in `netlify.toml` the config validation at the beginning of the build will fail with an error message.
```
11:32:34 AM: Failed during stage 'Reading and parsing configuration files': 
When resolving config file /opt/build/repo/netlify.toml:
Configuration property edge_functions[1].path must be a valid path.

Invalid syntax

  [[edge_functions]]
  path = "hello/*"

Valid syntax

  [[edge_functions]]
  path = "/hello"
  function = "hello"

: exit status 1
```

2. If the slash is missing in the in-source-config then TypeScript will warn about that.

So it is pretty unlikely that this error comes from invalid user input, but more likely from an integration that generates a pattern without a slash. So I think a user will in 90% not know what to do, no matter how we name it. 

I wonder if we should instead of validating the manifest should have validation for edge function declarations from integrations, so that we can point the user to `an integration was declaring an invalid edge function`

Reply:
> I wonder if we should in addition of validating the manifest have validation for edge function declarations from integrations, so that we can point the user to `an integration was declaring an invalid edge function`

I like this idea! This could give the user a more actionable step forward (remove the integration or contact the integration provider with a bug report). It would be great if an error message like this could provide `generator` info for the problematic function to help users figure out which integration to remove / contact with a bug report.